### ISSUE TITLE
[BUG] microns to dva plotting error

### ISSUE THREAD

**Issue by ju-w** [pulse2percept/pulse2percept#559]
**Describe the bug**
Bounding box and optic disc location should be mirrored at the x-axis when changing from microns to dva.

**To Reproduce**
```python
model = p2p.models.AxonMapModel(
    loc_od=(10.5, 4.5),
    xrange=(-15, 5),  # dva
    yrange=(-2, 10),  # dva
    xystep=0.25,)

model.build()

fig, axs = plt.subplots(1,2)
model.plot(use_dva=False, ax=axs[0])
model.plot(use_dva=True, ax=axs[1])
fig.set_constrained_layout(True)

```

**Screenshots**
![p2p](https://github.com/pulse2percept/pulse2percept/assets/22564375/cf2d55cf-ad3f-4cee-a1f7-441649732000)

**Comment by jgranley** [comment]
Thanks for pointing this out. I also believe it should be flipped along the y axis too (the blind spot from optic disc should be in the left visual field for the right eye). 

I'm adding this to the 0.9 milestone and the cortex project, where model plotting has already been reworked to facilitate cortical models. 

Also just a note: this bug is just in the plotting, all of the models output still function as they should.

**Comment by jgranley** [comment]
Closed with #578 , merged into cortex, will be in master soon
